app/src/main/java/com/android/solvit/shared/model/request/ServiceRequest.kt

```diff
@@ -12,15 +12,15 @@ import com.android.solvit.shared.ui.theme.STARTED_color
 import com.google.firebase.Timestamp
 
 data class ServiceRequest(
-    val uid: String,
-    val title: String,
-    val type: Services,
-    val description: String,
-    val userId: String,
+    val uid: String = "",
+    val title: String = "",
+    val type: Services = Services.TUTOR,
```

Comment:
Why this is set by default Services.TUTOR

Reply:
Same question, but more generally, does it make sense to put default values for this fields? they did not have default values before because they are intended to be required

Reply:
Good point! The default value for Services.TUTOR was added to simplify object creation, especially when there is no immediate need to specify the service type.